lib/themes/dosomething/paraneue_dosomething/js/utilities/Globe.js

```diff
@@ -0,0 +1,12 @@
+import $ from 'jquery';
+import setting from './Setting';
+
+function init() {
+  const globeUpdateUrl = setting('dosomethingSetting.globeUrl');
```

Comment:
Are you sure you want to use `const`? [IE > 11](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/const).